packages/react-native-payments/package.json

```diff
@@ -74,18 +74,16 @@
     "gitHead": "b5608910319390f9773a9d42c3cc828e8e8a1d95",
     "dependencies": {
         "react-native-uuid": "^2.0.1",
-        "validator": "^13.9.0"
+        "validator": "^13.9.0",
+        "@rnw-community/platform": "^0.54.12",
+        "@rnw-community/shared": "^0.54.12"
     },
     "devDependencies": {
-        "@rnw-community/platform": "^0.54.12",
-        "@rnw-community/shared": "^0.54.12",
         "@types/validator": "^13.7.17",
         "react": "^18.2.0",
         "react-native": "^0.72.3"
     },
     "peerDependencies": {
-        "@rnw-community/platform": ">=0.49",
-        "@rnw-community/shared": ">=0.49",
```

Comment:
@vitalyiegorov I have also moved platform and shared packages from devDeps to deps, as this package's consumers have to explicitly those packages in order to use rn-payments package.